File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/BatWing.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item;

/* JADX INFO: loaded from: classes3.dex */
public class BatWing extends Item {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.item_bat_wing_name;
        this.idDescription = R.string.item_bat_wing_description;
        this.idImage = R.drawable.bat_wing;
        this.source.add(Integer.valueOf(R.string.dungeon_name_obsidian_mines));
        this.price = 3L;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/BatWing.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item

class BatWing : Item() {
    override fun configureProperties() {
        idName = R.string.item_bat_wing_name
        idDescription = R.string.item_bat_wing_description
        idImage = R.drawable.bat_wing
        source.add(R.string.dungeon_name_obsidian_mines)
        price = 3L
    }
}
